CMakeLists.txt

```diff
@@ -63,6 +64,11 @@ if (FMT_MASTER_PROJECT AND NOT CMAKE_BUILD_TYPE)
               "CMAKE_C_FLAGS used) Debug Release RelWithDebInfo MinSizeRel.")
 endif ()
 
+set(FMT_CAN_MODULE OFF)
+if (CMAKE_CXX_STANDARD GREATER 17 AND NOT MSVC)
+  set(FMT_CAN_MODULE ON)
+endif ()
```

Comment:
Most compiler versions don't support modules so this should be disabled by default, not enabled by default.

Reply:
This is just a guard for the dependent option which will hide the option if the new module system is not available. If this is `TRUE`, `FMT_MODULE` will be the default value which is `FALSE`, as in the following code segment. Or otherwise, the option will be hidden completely from the user.